src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/ImplementIDisposableCorrectlyTests.cs

```diff
@@ -591,7 +630,98 @@ public class B : A
     }
 }|]
 ",
-            GetCA1063CSharpFinalizeOverrideResultAt(29, 14, "C"));
+            GetCA1063CSharpFinalizeImplementationResultAt(31, 6, "C", "Finalize"));
+        }
+
+        [Fact, WorkItem(1950, "https://github.com/dotnet/roslyn-analyzers/issues/1950")]
```

Comment:
Probably following test should be added to make sure, that the analyzer is relaxed and does not return warning, when there is no destructor. Just like in previous version.

```csharp
        [Fact, WorkItem(1950, "https://github.com/dotnet/roslyn-analyzers/issues/1950")]
        public void CSharp_CA1063_FinalizeOverride_NoDiagnostic_FinalizeNotOverriden()
        {
            VerifyCSharp(@"
using System;

public class B : IDisposable
{
    public void Dispose()
    {
        Dispose(true);
        GC.SuppressFinalize(this);
    }

    ~B()
    {
        Dispose(false);
    }

    protected virtual void Dispose(bool disposing)
    {
    }
}

[|public class C : B
{
    protected override void Dispose(bool disposing)
    {
        base.Dispose(disposing);
    }
}|]
");
        }
```